examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -99,6 +99,19 @@ databases: []
 #      "*": "*"
 databaseResources: []
 
+################################################################
+# Values that must be provided for Kubernetes Discovery
+################################################################
+
+# kubernetesDiscovery controls configuration for discovery service if it's enabled.
+# Default value will try to discovery all apps, use needs to override it to limit
+# scope of the discovered apps.
+kubernetesDiscovery:
```

Comment:
Non-blocking question: in the future we want to also add discovery support for other thinks like cluster discovery or database discovery. Would it make more sense to have a nested structure or am I over-thinking this?

e.g. 
```yaml
discovery:
  kubernetes: {}
  aws: {}
  azure: {}
  gcp: {}
  nextFancyThing: {}
```

Reply:
I would think about it when we get close to actually implementing those things (who knows how it will be handled in the end, maybe dynamic discovery config will take care of it?). If we really need we can always later coalesce values into single var if we want to change the structure.